frontend/src/components/core/Block/Block.tsx

```diff
@@ -559,6 +563,17 @@ class Block extends PureComponent<Props> {
         )
       }
 
+      case "cameraImageInput": {
+        return (
+          <CameraImageInput
+            element={node.element.cameraImageInput as CameraImageInputProto}
+            widgetMgr={widgetProps.widgetMgr}
+            uploadClient={this.props.uploadClient}
+            disabled={false}
```

Comment:
why do we pass in disabled when it's always going to be false? I don't think we need to propagate this?